template/en/default/account/prefs/mfa.html.tmpl

```diff
@@ -16,11 +23,15 @@
   [% RETURN %]
 [% END %]
 [% IF user.cryptpassword == '*' %]
-  <input type="hidden" name="mfa_action" id="mfa-action" value="">
+  <input type="hidden" name="mfa_action" id="mfa-action" value="" data-nopassword="true">
   <p>
     Two-factor Authentication is not available on your account because you are
     using an external authentication provider.
   </p>
+  <p>
+    Clicking "Reset Password" will cause [% terms.Bugzilla %] to send you an
+    email which will allow you to create a password.
```

Comment:
this message should describe why a "reset password" is required here, and what the implications will be.  in particular if mfa-required means they will no longer be able to authenticate with github that needs to be crystal clear here.